packages/rpc/src/client.ts

```diff
@@ -89,13 +113,18 @@ export class RPCClient extends EventEmitter {
 	) => boolean;
 
 	private endpoint = 'https://discord.com/api';
+
 	private readonly transport: Class<typeof transports[keyof typeof transports]>;
-	private readonly _expecting: Map<string, { resolve: (value: unknown) => void; reject: (reason?: Error) => void }>;
+
+	private readonly _expecting: Map<
+		string,
+		{ error: RPCError; reject(reason?: Error): void; resolve(value: unknown): void }
+	>;
+
 	private _connectPromise: Promise<unknown> | undefined;
-	private readonly _subscriptions: Map<string, ({ shortcut }: { shortcut: string }) => void>;
 
 	/**
-	 * @param {RPCClientOptions} [options] Options for the client.
+	 * @param options - Options for the client.
 	 * You must provide a transport
 	 */
 	public constructor(options: RPCClientOptions = {}) {
```

Comment:
If "You must provide a transport", why is the parameter optional?

Reply:
It's not optional,

Reply:
`You must provide a transport` can be removed from the tsdoc then